src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="page">
+      <div data-qa="card" class="page__card card">
+        <div class="card__image"></div>
+
+        <div class="card__info info-card">
```

Comment:
```suggestion
        <div class="card__info">
```
it's just an element even if contains nested elements
P.S. This wrapper seems redundant to me - just add a bigger margin to the Price block